fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1309,6 +1327,20 @@ public long saveInsertOverwriteJobs(DataOutputStream dos, long checksum) throws
         return checksum;
     }
 
+    public long saveStarMgrMeta(DataOutputStream dos, long checksum) throws IOException {
+        if (Config.integrate_starmgr) {
```

Comment:
You should not dependent on the config. Config can be changed, but you must read the image if you have written it.

Reply:
And more, you should add some message to the checksum. If the file has been damaged, it can be identified as soon as possible.

Reply:
yes, i noticed that no matter intergrate_starmgr true or false, meta should be written to the image.

Reply:
i will write a checksum, but right now it's not changed like some other modules.